packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -576,7 +576,7 @@ export class GarbageCollector implements IGarbageCollector {
 		const gcResult = runGarbageCollection(gcData.gcNodes, ["/"]);
 		// Get all referenced nodes - References in this run + references between the previous and current runs.
 		const allReferencedNodeIds =
-			this.findAllNodesReferencedBetweenGCs(gcData, this.gcDataFromLastRun, logger) ??
+			this.findAllNodesReferencedBetweenGCs(gcData, this.gcDataFromLastRun, fullGC, logger) ??
```

Comment:
I actually have a change in progress where I am adding fullGC to the logger used during GC. So, all logs will have it.

Reply:
Oh ok.  Let's discuss whether to move forward with this PR or abandon it